Add download user list button

Bastianleaf opened the pull request:
Se agrega botón de descarga de lista de usuarios con su información y permisos asociados (grupos asociados)

Bastianleaf committed:
Add changelist with download button

Bastianleaf committed:
Add get_all_users_list function

Bastianleaf committed:
Add DownloadUserList url, view and tests

Bastianleaf committed:
Add DownloadUserList url, view and tests

felipinbombin commented on localinfo/urls.py:
```diff
@@ -10,5 +10,7 @@
     url(r'^faq/$', FaqHTML.as_view(), name='faq'),
-    url(r'^opProgramList/$', OPProgramList.as_view(), name='opProgramList')
+    url(r'^opProgramList/$', OPProgramList.as_view(), name='opProgramList'),
+    url(r'^downloadUserData/$', DownloadUserList.as_view(), name='downloaduserlist')
```
estás dos urls deben tener aplicado `login_required`, sin esto son urls públicas que cualquier persona puede acceder. Esto también se debería aplicar a la url `faq`.

Bastianleaf replied:
Corregido

Bastianleaf committed:
Merge branch 'master' of https://github.com/SmartcitySantiagoChile/fondefVizServer into feature/DownloadUserList

Bastianleaf committed:
Add login_required to a missing localinfo urls

Bastianleaf committed:
Update get all_user_list:
  - Rename to create_csv_format_users_list.
  - Create dynamically a csv format with all existing permissions.
  - Return a list of list with csv format and header included.
  - Fix tests

Bastianleaf committed:
Update DownloadUserList view with create_csv_format_users list

Bastianleaf commented:
Se agregan las mejoras solicitadas.

felipinbombin committed:
replace login_required with staff_member_required because urls are administration actions

felipinbombin committed:
add postproducts permission to es api views related

felipinbombin committed:
add 'Permiso ' string previous to permission name

felipinbombin committed:
fixed test
